readers/influxdb/messages_test.go

```diff
@@ -50,7 +50,9 @@ var (
 )
 
 func TestReadSenml(t *testing.T) {
-	writer := iwriter.New(client, repoCfg, true)
+	// Testing both async and sync
```

Comment:
This is a Reader test, no need to use both types of Consumers.

Reply:
Should I use sync or async consumer for this test?

Reply:
I used async